### DISCUSSION TITLE
Info only - IPV6 blocks rtsp camera access from Frigate and other software

### DISCUSSION THREAD

**Discussion by bluezclouds** [blakeblackshear/frigate#10571] [General]
Posting to save someone experiencing the same problem accessing a rtsp capable camera a few hours of troubleshooting...

I started with a fully working Frigate installation on Debian 12 Home Assistant Supervisor with a Dahua camera.  After disabling it for a couple of weeks, when the software was re-started it would not connect to the cameras and threw a username/password error in the log.   This happened even though the username and password were correct and no changes had been made to the frigate.yaml file. 

It turned out that Debian had pulled an IPV6 address from somewhere, (despite IPV6 being disabled in both the sysctl.conf and on the router running the DHCP server) and that was preventing the rtsp string from working correctly.

After disabling IPV6 in the kernel config on the server and making no other changes the camera began working again.

**Comment by tcsweetser** [comment]
Appalling that cameras are not working with IPv6 correctly, given IoT is supposed to really need IPv6 to even be useful.
